WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginBrowserActivity.java

```diff
@@ -70,6 +75,9 @@
         }
     }
 
+    @Inject
+    ViewModelProvider.Factory mViewModelFactory;
```

Comment:
Nitpick: We keep the `@Inject` statements on the same line for properties. That's something we started to do with FluxC and recently I think we were looking for a way to add it into the lint checks. Let's keep things consistent here :)

Reply:
Fixed